app/assets/stylesheets/poll.scss

```diff
@@ -1,18 +1,20 @@
 .poll_form {
```

Comment:
Renaming this class would introduce a lot of other code style errors and thus I think that this should happen in another PR.

```bash
grep -Rn "poll_form" app features spec
app/assets/stylesheets/new_styles/_poll.scss:1:.poll_form {
app/assets/stylesheets/color_themes/_color_theme_override.scss:29:  .poll_form .progress .bar { background-color: $main-color-dark; }
app/assets/templates/poll_tpl.jst.hbs:2:  <div class="poll_form">
features/desktop/post_with_a_poll.feature:49:      Then I should see a ".poll_form" within ".stream_element"
features/desktop/post_preview.feature:85:      Then I should see a ".poll_form" within ".stream_element"
features/step_definitions/post_with_poll_steps.rb:22:  page.should have_css('.poll_form input')
features/step_definitions/post_with_poll_steps.rb:23:  first(".poll_form input").click
```